pythonforandroid/recipe.py

```diff
@@ -965,20 +946,13 @@ def build_cython_components(self, arch):
                 info('First build appeared to complete correctly, skipping manual'
                      'cythonising.')
 
-            if 'python2' in self.ctx.recipe_build_order:
-                info('Stripping object files')
-                build_lib = glob.glob('./build/lib*')
-                shprint(sh.find, build_lib[0], '-name', '*.o', '-exec',
-                        env['STRIP'], '{}', ';', _env=env)
-
-            else:  # python3crystax or python3
-                info('Stripping object files')
-                shprint(sh.find, '.', '-iname', '*.so', '-exec',
-                        '/usr/bin/echo', '{}', ';', _env=env)
-                shprint(sh.find, '.', '-iname', '*.so', '-exec',
-                        env['STRIP'].split(' ')[0], '--strip-unneeded',
-                        # '/usr/bin/strip', '--strip-unneeded',
-                        '{}', ';', _env=env)
+            info('Stripping object files')
+            shprint(sh.find, '.', '-iname', '*.so', '-exec',
+                    '/usr/bin/echo', '{}', ';', _env=env)
```

Comment:
Looks like it was already like that, but now I'm getting curious why this is working, since:
```
ls /usr/bin/echo
ls: cannot access '/usr/bin/echo': No such file or directory
```

Reply:
I'm not sure how it would work, but i think the -ls option of find is what one would want to use.

Reply:
I've been testing  this from terminal, and it fails on the echo stage, because, As @AndreMiras pointed,  the echo command is not located at /usr/bin/echo, actually should be /bin/echo or echo...but even changing this  the result is exactly the same...so I suspect that the combination of this two commands cause the issue...so the @tshirtman's proposal maybe a  solution...can you give me any clue about of how to apply the find command with -ls, filtering  by  extension ".so" and printing line by line?

Reply:
I suppose this line just wasn't doing anything on systems where the path to echo is wrong. `sh` doesn't actually raise an error if this happens, it would just do nothing.

Reply:
Also, you can just take out the -exec stuff, you can still extract a list of files from the result (good enough for printing). Or just remove the print entirely, it isn't really important.

Reply:
In the next commit, I was planning to restore this lines at his previous state (to introduce python2legacy recipe) and remove entirely this print statement as you suggested, but now, I'm thinking to go a little further...perhaps we can create a new method  like **strip_cython_files** into TargetPythonRecipe (as you do with **create_python_bundle**)?